frontend/src/pages/Dashboard/components/RecordDrawer/RecordDrawer.module.scss

```
@@ -247,6 +257,69 @@
+
+  pre {
+    margin-top: var(--s2);
+    padding: var(--s3);
+    border-radius: var(--r-sm);
+    background: var(--fill);
+    color: var(--ink-2);
+    font-family: inherit;
+    font-size: 12px;
+    line-height: 1.7;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)